src/main/java/winstone/ServerConnectorFactory.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2003-2006 Rick Knowles <winstone-devel at lists sourceforge net>
+ * Distributed under the terms of either:
+ * - the common development and distribution license (CDDL), v1.0; or
+ * - the GNU Lesser General Public License, v2.1 or later
+ */
+package winstone;
+
+import org.eclipse.jetty.server.ForwardedRequestCustomizer;
+import org.eclipse.jetty.server.HttpConfiguration;
+import org.eclipse.jetty.server.HttpConnectionFactory;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.server.ServerConnector;
+import org.eclipse.jetty.util.ssl.SslContextFactory;
+import winstone.cmdline.Option;
+
+import java.util.Map;
+
+public class ServerConnectorFactory {
```

Comment:
I would prefer using Builder pattern and not holding the reference to the ServerConnector

Reply:
Thank you for the hint - that's better for this case.